.github/workflows/development-environment.yml

```diff
@@ -40,9 +40,10 @@ jobs:
         # Xcode CLI & brew are already installed, thus, no need to call xcode-select install
         # XXX: Can the Brew set up be cached?
         # Sometimes, brew needs to be updated before brew bundle would work
-        # If you want to speed up the CI for your PR, you can use --no-upgrade and HOMEBREW_NO_AUTO_UPDATE=1
+        # After installing Docker we can sure that is properly initialized on Mac
         run: |
-          brew bundle || brew update && brew bundle
+          HOMEBREW_NO_AUTO_UPDATE=1 brew bundle -q --no-upgrade || brew update && brew bundle
+          SENTRY_NO_VENV_CHECK=1 ./scripts/do.sh init_docker
```

Comment:
Not implementing it as a make target since I don't want anyone calling the target. I want less make target fatigue.